### PR TITLE
update statistician docker image

### PR DESCRIPTION
**Pull request by NikitaGandhi**

### CODE REVIEW

File: statistician/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,8 @@
 --extra-index-url https://packages.dea.ga.gov.au/
 
-datacube[performance,s3]==1.8.5
-aiobotocore[awscli,boto3]==1.3.3
```
Comment (alexgleith) [inline statistician/requirements.txt:4]: Unpinning this is dangerous

File: statistician/requirements.txt [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
+datacube[performance,s3]
+aiobotocore[awscli,boto3]
 odc-stats
-hdstats==0.1.8.post1
```
Comment (alexgleith) [inline statistician/requirements.txt:6]: Why unpin here?

File: statistician/Dockerfile [hunk 3]
```diff
@@ -39,5 +39,10 @@ WORKDIR /tmp
 COPY --from=env_builder $py_env_path $py_env_path
 ENV PATH="${py_env_path}/bin:${PATH}"
 
+# override hdstats - install faster version from dea packages
+RUN echo "Patching python env - hdstats" \
+  && pip install --force-reinstall \
+  --extra-index-url="https://packages.dea.ga.gov.au" hdstats==0.1.8.post1
+
```
Comment (alexgleith) [inline statistician/Dockerfile:46]: I understand needing to do this in Sandbox image, but in this image, can't se just require it with a constraint? Was that not working?

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: statistician/requirements.txt (hunk 1 above)
Comment (NikitaGandhi) [reply statistician/requirements.txt:4]: it is pined under constraints file

File: statistician/requirements.txt (hunk 2 above)
Comment (NikitaGandhi) [reply statistician/requirements.txt:6]: by default hdstats install 0.2.1 even you pin it here so I have to add that additional step

File: statistician/requirements.txt (hunk 1 above)
Comment (alexgleith) [reply statistician/requirements.txt:4]: Still I think it needs to be pinned here, due to how we re-create the constraints.

File: statistician/requirements.txt (hunk 2 above)
Comment (alexgleith) [reply statistician/requirements.txt:6]: Strange. Ok!

File: statistician/requirements.txt (hunk 1 above)
Comment (NikitaGandhi) [reply statistician/requirements.txt:4]: done :)

**Review by alexgleith — APPROVED** [review]

**Pull request merged**